### PR TITLE
CASMINST-5502 IUF: Add support for loading repositories 

### PR DESCRIPTION
**Pull request by rnoska-hpe**
# Description

Add Install Upgrade Framework (IUF) support for loading one or more Nexus type 'yum' or 'raw' repositories.

# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by dzou-hpe — APPROVED** [review]

**Review by heemstra — APPROVED** [review]

**Review by nrockershousen — APPROVED** [review]

File: workflows/iuf/operations/nexus-setup/README.md [hunk 1]
```diff
@@ -73,3 +73,29 @@ argo -n argo submit --from workflowtemplate/nexus-docker-upload-template \
...
+
+```bash
+argo -n argo template create nexus-rpm-upload-template.yaml
+```
+
+The example below shows how to submit the template using the current required parameters. This example requires the existence
+of the product at `$PRODUCTS_DIR`/`$PRODUCT` and requires the `$NEXUS_SETUP_IMAGE` to be present in Nexus.
+
+```bash
+PRODUCTS_DIR=/admin/rnoska/argo-nexus/nexus-upload/products
```
Comment (nrockershousen) [inline workflows/iuf/operations/nexus-setup/README.md:91]: Do we want to substitute `rnoska` with something more generic here?

File: workflows/iuf/operations/nexus-setup/README.md [hunk 2]
```diff
@@ -73,3 +73,29 @@ argo -n argo submit --from workflowtemplate/nexus-docker-upload-template \
...
+PRODUCT=cos-2.5.38-20221024172946
+NEXUS_SETUP_IMAGE=artifactory.algol60.net/csm-docker/unstable/cray-nexus-setup:0.8.1-20221110002429_3de03c8
+
```
Comment (rustydb) [inline workflows/iuf/operations/nexus-setup/README.md:94]: Break this up into two blocks, so the command below is easier to copy-paste.

**Review by rustydb — CHANGES_REQUESTED** [review]

File: workflows/iuf/operations/nexus-setup/README.md (hunk 1 above)
Comment (rnoska-hpe) [reply workflows/iuf/operations/nexus-setup/README.md]: I've removed these from the example.

File: workflows/iuf/operations/nexus-setup/README.md (hunk 2 above)
Comment (rnoska-hpe) [reply workflows/iuf/operations/nexus-setup/README.md]: I've removed these from the example.

**Review by rustydb — APPROVED** [review]

**Pull request merged**